sagemaker-triton/fil_ensemble/README.md

```diff
@@ -0,0 +1,19 @@
+# XGBoost model inference pipeline with NVIDIA Triton Inference Server on Amazon SageMaker
+
+In this example we show an end-to-end GPU-accelerated fraud detection example making use of tree-based models like XGBoost. In the first notebook [1_prep_rapids_train_xgb.ipynb](1_prep_rapids_train_xgb.ipynb) we demonstrate GPU-accelerated tabular data preprocessing using RAPIDS and training of XGBoost model for fraud detection on the GPU in SageMaker. Then in second notebook [2_triton_xgb_fil_ensemble.ipynb](2_triton_xgb_fil_ensemble.ipynb) we walkthrough the process of deploying data preprocessing + XGBoost model inference pipeline for high throughput, low-latency inference on Triton in SageMaker. 
+
+## Steps to run the notebooks
+1. Launch SageMaker **notebook instance** with `g4dn.xlarge` instance.
```

Comment:
Clarify whether it is possible to run in Studio instead, or whether a notebook instance is required.

Reply:
Added blurb to point out it can be run on Studio but we will focus on steps that show running on a notebook instance